### DISCUSSION TITLE
Order by of Date in Foreign Table not sorting the list

### DISCUSSION THREAD

**Discussion by rmrbytes** [supabase/supabase#11859] [Questions]
I have the following relationship

A publication table with
```
pubs {
   id : uuid
   title :Text
   ...
   updated_at: timestamptz
}
```

A subscription table with 
```
subs {
   id
   user_id
   pubs_id // this is FK
     ...
}
```

My fetch is

```
// first retrieve widgets already subscribed by user  
      let { data: subs, error } = await supabase
        .from('subs')
        .select(`
              id,
              user_id,
              pubs_id,
              pubs!inner(*)
            `)
        .eq('user_id', user.id)
        .order('updated_at', {foreignTable: "pubs",  ascending:false})  // need sorted by updated_at in pubs table
```

Yet I find that the data does not appear sorted. Even setting the `{ascending: true}` makes no difference to the order.

**Comment by rmrbytes** [comment]
Some of the discussions seems to suggest that it is a Postgresql limitation of having a sort on the joined table attribute. For now, I sort the array in the client (possible for my use-case) before feeding it to the UI

**Comment by steve-chavez** [comment]
If pubs-sub is a many-to-one or one-to-one relationship the order would work with this feature:

https://supabase.com/blog/postgrest-11-prerelease#order-by-related-tables

**Comment by rmrbytes** [comment]
Thanks @steve-chavez. The subs has the pubs id as a FK as mentioned. Just to be clear, the primary access is via the subs table because there is a built-in where clause (RLS).  Essentially filtering all subs of the logged user but displaying them on the updated field of the pubs table. 

Just FYI, I am using v 2.2.0 of supabase-js.
